.devcontainer/Dockerfile

```diff
@@ -3,12 +3,15 @@ FROM alpine:3.18
 ENV TZ UTC
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 
+COPY ./xdebug.ini /etc/php81/xdebug.ini
+
 RUN apk add --no-cache \
 	tzdata \
 	apache2 php-apache2 \
 	php php-curl php-gmp php-intl php-mbstring php-xml php-zip \
 	php-ctype php-dom php-fileinfo php-iconv php-json php-opcache php-openssl php-phar php-session php-simplexml php-xmlreader php-xmlwriter php-xml php-tokenizer php-zlib \
 	php-pdo_sqlite php-pdo_mysql php-pdo_pgsql \
+    php81-pecl-xdebug \
```

Comment:
We might consider moving everything to PHP 8.2 for the development container

Reply:
Let me make another PR for that

Reply:
Hum, `composer` still requires `php81` explicitly https://pkgs.alpinelinux.org/package/v3.18/community/x86_64/composer , so let's wait for it